apps/web/src/utils/isPoolTickInRange.ts

```diff
@@ -1,6 +1,7 @@
 import { Pool } from '@pancakeswap/v3-sdk'
 
-const isPoolTickInRange = (pool: Pool, tickLower: number, tickUpper: number) => {
+const isPoolTickInRange = (pool: Pool | undefined, tickLower: number, tickUpper: number) => {
```

Comment:
Can we also add the type to `tickLower` and `tickUpper`?
```suggestion
const isPoolTickInRange = (pool: Pool | undefined, tickLower: number | undefined, tickUpper: number | undefined) => {
```